services/cards/src/actions/Cards.tsx

```diff
@@ -37,23 +79,19 @@ export function downloadCards(): ((dispatch: Redux.Dispatch<any>) => void) {
             return card;
           }));
         });
-        dispatch(cardsUpdate(cards));
-        dispatch(cardsFilter(store.getState().cards.data, store.getState().filters));
-        dispatch(filtersCalculate(store.getState().cards.filtered));
+        resolve({cards, translations});
       },
-      key: store.getState().filters.source.current.split(':')[1],
+      key,
       parseNumbers: true,
       postProcess: (card: CardType) => {
         // TODO parse / validate / clean the object here. Use Joi? Expose validation errors to the user
-        // Note that we can't make too many assumptions about the data coming in if we want this to work with
-        // multiple games... unless each theme has its own validation schema!
         // Note: also have to be careful about the translations settings sheet
         // Note: doesn't yet have access to sheet name, that's assigned in callback
         return card;
       },
       simpleSheet: true,
     });
-  };
+  });
 }
 
 export interface CardsLoadingAction extends Redux.Action {
```

Comment:
Mind quickly moving these to an `ActionTypes` file to be consistent with all our other services?